package/MDAnalysis/analysis/hole.py

```diff
@@ -356,6 +362,7 @@ def plot(self, **kwargs):
         ax.set_ylabel(r"HOLE radius $r$ ($\AA$)")
         if kw['label']:
             ax.legend(loc="best")
+        return ax
```

Comment:
Thanks, will fix.

Comments like yours will go some way to de-crudding some of the older code...